extension/types/squire.d.ts

```diff
@@ -1,9 +1,12 @@
+import * as forge from 'node-forge';
+
 export as namespace SquireClass;
 
 declare global {
   interface Window {
     // @ts-ignore
     Squire: SquireClass;
+    forge: typeof forge;
   }
 }
 
```

Comment:
Also this is wrong to add forge into squire definitions. Let's remove it and file an issue for Ioan to have a look later.

Reply:
okay sir, I'll remove it - I just converted most of it from any to unknown but there are still some that need to have proper types but I can't convert them just yet.

Reply:
That's ok. You can leave these for another PR in the future.